app/com/linkedin/drelephant/tuning/hbt/ParameterGenerateManagerHBT.java

```diff
@@ -83,6 +83,7 @@ private String generateParamSet(List<TuningParameter> tuningParameters, JobDefin
         .eq(JobExecution.TABLE.job, job)
         .ne(JobExecution.TABLE.executionState, ExecutionState.CANCELLED)
         .ne(JobExecution.TABLE.executionState, ExecutionState.FAILED)
+        .ne(JobExecution.TABLE.executionState, ExecutionState.NOT_STARTED)
```

Comment:
What is this change for ?

Reply:
Since we changed some execution to NOT_STARTED (which is the same state as Failed & Cancelled) . This has to do nothing for PIG HBT .